contracts/core/EntryPoint.sol

```diff
@@ -621,43 +621,26 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
         );
         address paymaster = opInfo.mUserOp.paymaster;
         bool success;
+        uint256 contextOffset;
         uint256 contextLength;
+        uint256 returnDataSize;
         assembly ("memory-safe") {
-            //call and return 3 first words: offset, validation, context-length
-            success := call(gas(), paymaster, 0, add(validatePaymasterCall, 0x20), mload(validatePaymasterCall), freePtr, 96)
+            success := call(gas(), paymaster, 0, add(validatePaymasterCall, 0x20), mload(validatePaymasterCall), freePtr, 2112)
```

Comment:
where this number came from? also, we don't need to copy always the worst case, as it should be configurable (and not in the solidity code, but in bundler